CourseGrab/Models/Section.swift

```diff
@@ -64,6 +64,27 @@ struct Section: Codable {
     let status: Status
     let subjectCode: String
     let title: String
+    
+    func getSectionByTimezone() -> String {
+        if let index = section.lastIndex(of: " ") {
+            let timeIndexStart = section.index(after: index)
+            let sectionString = String(section[..<timeIndexStart])
+            let timeString = String(section[timeIndexStart...])
+            
+            let dateFormatter = DateFormatter()
+            dateFormatter.dateFormat = "h:mma"
+            dateFormatter.timeZone = TimeZone(abbreviation: "EST")
```

Comment:
I'm always confused about `EST` vs `EDT` etc, the default should probably just be `ET` for Eastern Time that way we don't run into any confusion about daylight savings. 🌤️

Reply:
The fact that it's EST part of the year and EDT the other part is probably the dumbest thing I've ever heard. Anyway, the Apple docs discourage using the `abbreviation` initializer:

 > In general, you are discouraged from using abbreviations except for unique instances such as “GMT”. Time Zone abbreviations are not standardized and so a given abbreviation may have multiple meanings—for example, “EST” refers to Eastern Time in both the United States and Australia

So maybe use `TimeZone(name:)` instead and using `Eastern Time` like Mat said. Strangely, it seems like the name for EST and EDT are both `"America/New_York"`, so maybe Swift is smart and does the daylight savings conversion for you?